support-data-rest/src/main/resources/facet.properties

```diff
@@ -26,26 +26,22 @@ facet.limit=5
 
 facet.crossref.category_facet.label=Category
 facet.crossref.category_facet.allowmultipleselection=true
+facet.crossref.category_facet.limit=-1
 
-facet.taxonomy.reviewed.label=Swiss-Prot
-facet.taxonomy.reviewed.allowmultipleselection=false
-facet.taxonomy.reviewed.value.true=reviewed entries in UniprotKB
-facet.taxonomy.reviewed.value.false=without reviewed entries in UniprotKB
+facet.keyword.category.label=Category
```

Comment:
Should it be Categories? I really don't know, u know better, but i thought to mention